juego.py

```diff
@@ -37,21 +37,50 @@ def __init__(self, level, fuente):
         self.expresion = fuente.render(self.primero + simbolo[operador] + self.segundo + " = ? ",True,(255,0,0))
         self.resultado = str(eval(self.primero+operacion[operador]+self.segundo))
         self.vida = 0
-                                           
+        
+        no_repeat_check_x = []
+        no_repeat_check_y = []
+        list_y = [int(sx(-90)), int(sx(-80)), int(sx(-60)), int(sx(-50)), int(sx(-40)), int(sx(-30)),
+                int(sx(-20)), int(sx(-10)), int(sx(-0))]
+        list_x = [int(sx(100)), int(sx(200)), int(sx(300)), int(sx(400)), int(sx(500)), int(sx(600)),
+                int(sx(700)), int(sx(800)), 
```

Comment:
`juego.py:46:17: E128 continuation line under-indented for visual indent`
`juego.py:46:44: W291 trailing whitespace`